internal/drand-cli/dkg_cli.go

```diff
@@ -259,13 +259,7 @@ func parseInitialProposal(c *cli.Context) (*drand.FirstProposalOptions, error) {
 
 	// if a custom time hasn't been added, we set a one based on the period
 	// to make the tests a little more predictable
-	var genesisTime time.Time
-	if c.IsSet(transitionTimeFlag.Name) {
-		genesisTime = time.Now().Add(c.Duration(transitionTimeFlag.Name))
-	} else {
-		defaultRoundsToWait := period * 3
-		genesisTime = time.Now().Add(defaultRoundsToWait)
-	}
+	genesisTime := time.Now().Add(c.Duration(transitionTimeFlag.Name))
```

Comment:
For a CLI user, I think it's easier to think in duration than exact time. Also it's slightly less error prone in case people stick in e.g. epoch millis instead of epoch seconds.